Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3.yml

```diff
@@ -1770,7 +1725,100 @@ tasks:
       {
         "position": {
           "x": 510,
-          "y": -280
+          "y": -470
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "78":
+    id: "78"
+    taskid: cd52a989-d59a-4204-831c-60f8724e6a20
+    type: playbook
+    task:
+      id: cd52a989-d59a-4204-831c-60f8724e6a20
+      version: -1
+      name: Cortex XDR Alerts Handling
+      description: "This playbook is used to loop over every alert in a Cortex XDR incident. \nSupported alert categories:\n- Malware\n- Port Scan"
+      playbookName: Cortex XDR Alerts Handling
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "79"
+    scriptarguments:
+      alert_id:
+        complex:
+          root: PaloAltoNetworksXDR.Incident.alerts
+          accessor: alert_id
+      incident_id:
+        complex:
+          root: PaloAltoNetworksXDR
+          accessor: Incident.incident_id
+    separatecontext: false
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+      forEach: true
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 340
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "79":
+    id: "79"
+    taskid: 3ba84f9e-aaf6-4bd3-8d22-cfc00499e075
+    type: condition
+    task:
+      id: 3ba84f9e-aaf6-4bd3-8d22-cfc00499e075
+      version: -1
+      name: Check for Cryptomining alert type
```

Comment:
For future alerts, lets change to something like should continue with incident investigation? or something more generic